.circleci/config.yml

```
@@ -0,0 +1,31 @@
+version: 2.1
+orbs:
+  pack: buildpacks/pack@dev:1dcd173
+workflows:
+  version: 2
+  main:
+    jobs:
+      # This is added just to limit the number of runs, given that this is a sample
+      - request-testing:
+          type: approval
+      - pack/build:
+          working-directory: apps/java-maven
+          image-name: cnbs/circleci-sample
+          builder: 'paketobuildpacks/builder:base'
```

this would be our first departure from the vendor-neutral samples I think. However, I'm not sure if there's a way around it.

---

I don't see why we can't use our sample builders (alpine or bionic).

```suggestion
          builder: 'cnbs/sample-builder:bionic'
```